ui/src/App.tsx

```diff
@@ -63,15 +61,6 @@ const App = () => {
         icon={DeviceIcon}
         recordRepresentation={(record) => `${record.hostname}`}
       />
-      <Resource
-        name="zones"
-        list={ZoneList}
-        show={ZoneShow}
-        create={ZoneCreate}
-        icon={ZoneIcon}
-        recordRepresentation={(record) => `${record.name}`}
-      />
```

Comment:
What about adding Organizations?

Reply:
Was going to do in a separate PR but added to this.